kore/src/Kore/ASTVerifier/AttributesVerifier.hs

```diff
@@ -167,31 +163,31 @@ verifyNoHookedSupersort indexedModule axiom subsorts = do
             , "Its subsort:"
             , show . unparse $ Subsort.subsort sort
             , "Location in the Kore file:"
-            , show . prettyPrintAstLocation
-                $ locationFromAst (Subsort.supersort sort)
+            , show . prettyPrintAstLocation $
+                locationFromAst (Subsort.supersort sort)
```

Comment:
This is another significant departure from the usual style: putting `$` at the end of the line.